Source/IGListAdapterDataSource.h

```diff
@@ -60,7 +60,27 @@ NS_SWIFT_NAME(ListAdapterDataSource)
  adding the background view and maintaining its visibility.
  */
 - (nullable UIView *)emptyViewForListAdapter:(IGListAdapter *)listAdapter;
-
+    
+@optional
+    
+/**
+ Asks the datasource whether a list object should be movable as the result of interactive reordering.
+ 
+ @param listAdapter The list adapter sending this information.
+ @param sectionIndex The index of the section proposed to be moved.
+ @param index The index of the object proposed to be moved.
+ */
+- (BOOL)listAdapter:(IGListAdapter *)listAdapter canMoveObjectInSection:(NSInteger)sectionIndex atIndex:(NSInteger)index;
```

Comment:
Is there a way we could make this a property (defaulted to `NO`) on the adapter? Adding another (eventually) required method like this will be a pretty big pain to upgrade.

Another option, (which I like a lot more), can we add a `IGListAdapterMovingDelegate` or something onto `IGListAdapter`? Then we can make all the methods required _and_ not break any upgrades. Plus, if you don't use moving, it makes your data source implementation much cleaner.

Thoughts?

Reply:
Removing this entirely based on later feedback.